_posts/2020-12-24-how-to-start-with-foreman.md

```diff
@@ -91,8 +91,8 @@ hammer host create --name my-first-host \
   --location 'Default Location'
 ```
 
-Name is usually the FQDN of the host, but it can be also just a hostname without domain.
-The `managed` parameter tells Foreman, this will be unmanaged host. The build value
+Name is usually the FQDN of the host, but it can also be just a hostname without domain.
```

Comment:
Out of curiosity, why change this? You've split the infinitive with the change, not that it is a massive crime, but just curious, were you prompted by a text editor?

Reply:
google doc suggested it :-)